### PR TITLE
Feature development workflow.

### PR DESCRIPTION
**Pull request by duozersk**

### CODE REVIEW

File: docs/development_client_site.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+module to manage configuration updates coming from Falcon distribution.
+
+How to install Falcon feature?
+------------------------------
+Go to `/admin/modules` and install it.
+
+How to update Falcon feature?
+-----------------------------
+After updating the codebase to the new version of Falcon go to
+`/admin/config/development/configuration/distro` and import feature updates.
```
Comment (spleshka) [inline docs/development_client_site.rst:25]: Is there a drush command that does it? I was thinking about writing some hook_update which would update configs automatically.

File: docs/development_client_site.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+How to remove falcon feature?
```
Comment (spleshka) [inline docs/development_client_site.rst]: Is there (or can we provide) a way to automate this through hook_update or some drush command? There should be some automated upgrade path from old configs to new configs list. Maybe through hook_update or something like that.

File: docs/development_falcon.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+prefixed with `falcon_` string, for instance `falcon_example`.
+
+Packaged features should be placed into `modules/features` directory and
+prefixed with `falcon_feature_` string, for instance `falcon_feature_example`.
+
+Adding a new module
+-------------------
+When adding a new module to the Falcon distribution developer should decide if
+it is needed on every site consuming the distribution. If this is the case then
+it should be enabled using hook_update_N() in the `falcon_deploy` module.
```
Comment (spleshka) [inline docs/development_falcon.rst]: Shall it be added to `core.extensions.yml` or `falcon.info.yml`  as well?

File: docs/development_client_site.rst (hunk 1 above)
Comment (duozersk) [reply docs/development_client_site.rst:25]: There is the `config-distro-update` command. I would still start with the manual process and then improve it as we go. I will add a TODO to the docs about it.

File: docs/development_client_site.rst (hunk 2 above)
Comment (duozersk) [reply docs/development_client_site.rst]: I don't have a working process for it - we might try smth for it in the future, but I didn't have experience with it yet. Adding a TODO in docs.

File: docs/development_falcon.rst (hunk 3 above)
Comment (duozersk) [reply docs/development_falcon.rst]: Good catch! Added info about adding it as a dependency into falcon.info.yml file.

**Review by spleshka — APPROVED** [review]

**Pull request merged**